<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generator</class>
 <widget class="QWidget" name="Generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generation...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../share/mainResources.qrc">
    <normaloff>:/icons/images/new/finish.png</normaloff>:/icons/images/new/finish.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="lwLog"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/new/gtk-no.png</normaloff>:/icons/images/new/gtk-no.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../../share/mainResources.qrc">
         <normaloff>:/icons/images/new/stock_right.png</normaloff>:/icons/images/new/stock_right.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../share/mainResources.qrc"/>
 </resources>
 <connections/>
</ui>
